### DISCUSSION TITLE
[Proposition] Ajouter l'action personnalisée INVALID pour pouvoir mettre quand quelque chose de notre pull request est faux

### DISCUSSION THREAD

**Discussion by Mozinou2013** [INFOM126-Automated-Software-Engineering/2048-groupe-07#27] [New GitHub action]
Nom de l'action proposée : INVALID
Objectif de l'action (décrire ce qu'elle fait et pourquoi elle est nécessaire) : labeler les pull request qui contiennent quelque chose de faux
Fonctionnement (expliquer brièvement les étapes et les technologies utilisées) : Ajout d'un label sur la pull request

**Comment by ldelpier** [comment]
Quels seraient les fichiers ajoutés ?

**Reply by Mozinou2013** [reply]
Un fichier yml dans les workflows et une documentation si celle-ci peut être utile

Qu'en pensez-vous dans ce cas ?

**Reply by ldelpier** [reply]
Cela ne va pas faire la même chose que WontFix ?

**Reply by ldelpier** [reply]
Aussi non c'est une bonne idée

**Reply by Mozinou2013** [reply]
> Cela ne va pas faire la même chose que WontFix ?
Wontfix est là pour les pull requests qui comportent des erreurs et ces erreurs ne seront jamais régler
Invalid est pour les pull requests qui ont des erreurs et qui seront réglées

**Reply by ldelpier** [reply]
Parfait alors c'est bon pour moi

**Reply by Mozinou2013** [reply]
Merci
